src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/Details.cshtml

```diff
@@ -11,8 +11,8 @@
 
 <script src="@Url.Content("~/Scripts/jquery.unobtrusive-ajax.js")" type="text/javascript"></script>
 <script>
-    var onArchiveSuccess = function() {
-        alert('Tournament was successfully archived');
+    var onSuccess = function() {
+        alert('@Model.ChangingStateMessage'.concat(' now.'));
```

Comment:
Why 'now' part is here and everything else is somewhere?

Reply:
Ok, changing.